tests/libvmifact/architecture.go

```diff
@@ -19,12 +19,51 @@
 
 package libvmifact
 
-var architecture string
+import "kubevirt.io/kubevirt/pkg/libvmi"
+
+const (
+	// required to start qemu on ARM with UEFI firmware
+	// https://github.com/kubevirt/kubevirt/pull/11366#issuecomment-1970247448
+	armMinimalBootableMemory = "128Mi"
```

Comment:
nit: Just to keep the code clean, I'd add these comments to your commit message. Then, if needed, developer can check the commit message to see why this variable was introduced in this PR.